webhook/url_notifier.go

```diff
@@ -141,3 +163,46 @@ func (n *URLNotifier) send(event *livekit.WebhookEvent) error {
 type logAdapter struct{}
 
 func (l *logAdapter) Printf(string, ...interface{}) {}
+
+func logFields(event *livekit.WebhookEvent) []interface{} {
+	fields := make([]interface{}, 0, 20)
+	fields = append(fields,
+		"event", event.Event,
+		"id", event.Id,
+		"webhookTime", event.CreatedAt,
+	)
+
+	if event.Room != nil {
+		fields = append(fields,
+			"room", event.Room.Name,
+			"roomID", event.Room.Sid,
+		)
+	}
+	if event.Participant != nil {
+		fields = append(fields,
+			"participant", event.Participant.Identity,
+			"pID", event.Participant.Sid,
+		)
+	}
+	if event.EgressInfo != nil {
+		fields = append(fields,
+			"egressID", event.EgressInfo.EgressId,
+			"status", event.EgressInfo.Status,
+		)
+		if event.EgressInfo.Error != "" {
+			fields = append(fields, "error", event.EgressInfo.Error)
+		}
+	}
+	if event.IngressInfo != nil {
+		fields = append(fields,
+			"ingressID", event.IngressInfo.IngressId,
+		)
+		if event.IngressInfo.State != nil {
+			fields = append(fields, "status", event.IngressInfo.State.Status)
```

Comment:
guess ingress and egress cannot co-exist, so field collision of `status` and `error` cannot happen, but wondering if it makes sense to just call them `egressStatus`/`egressError`. Just a thought, but `status`/`error` works fine.

Reply:
right, they are not able to coexist.. the goal here is to standardize our logging fields to make it easier for searches/etc.

Reply:
yep, makes it easier to search, good point.